bai8.js

```diff
@@ -0,0 +1,12 @@
+const countOccurrences = (list, target) => {
+  const item = list.split("");
```

Comment:
It seems that you are taking a string instead of an array `8. Write a function countOccurrences that takes an array of strings and a target string, and returns the number of times the target string appears in the array.`